logging/channels_handlers.rst

```diff
@@ -180,15 +180,15 @@ by type-hinting your service arguments with the following syntax:
 ``Psr\Log\LoggerInterface $<camelCased channel name> + Logger``. The ``<channel>``
 must have been :ref:`predefined in your Monolog configuration <monolog-channels-config>`.
 
-For example to inject the service related to the ``foo_bar`` logger channel,
+For example to inject the service related to the ``foo`` logger channel,
 change your constructor like this:
 
 .. code-block:: diff
 
     -     public function __construct(LoggerInterface $logger)
-    +     public function __construct(LoggerInterface $fooBarLogger)
+    +     public function __construct(LoggerInterface $fooLogger)
```

Comment:
What about keeping this but instead declare the foo_bar ?
So that we see the snake case to came case conversation in the doc

Reply:
Thanks for the suggestion, it's done.